src/patternfly/layouts/Page/styles.scss

```diff
@@ -4,97 +4,86 @@
 $nav: "pf-c-nav";
 
 .pf-l-page {
-
-  /* stylelint-disable */
-
   // Header
-  --pf-l-page__header--PaddingBottom:                       var(--pf-global--spacer--md);
-  // Hardcode header height base on design spec
-  --pf-l-page__header--MinHeight:                           pf-size-prem(110px);
+  --pf-l-page__header--PaddingBottom: var(--pf-global--spacer--md);
+  --pf-l-page__header--MinHeight: pf-size-prem(110px); // Hardcode header height base on design spec
 
   // Header brand
-  --pf-l-page__header-brand--PaddingTop:                    var(--pf-global--spacer--md);
-  --pf-l-page__header-brand--PaddingBottom:                 var(--pf-global--spacer--md);
-  --pf-l-page__header-brand--PaddingLeft:                   var(--pf-global--spacer--xl);
-  --pf-l-page__header-brand--PaddingRight:                  var(--pf-global--spacer--xl);
-  --pf-l-page__header-brand--PaddingBottom--sm:             var(--pf-global--spacer--md);
-  --pf-l-page__header-brand--BackgroundColor--md:           var(--pf-global--BackgroundColor--dark-transparent-100);
-  // Hardcode header brand width based on design spec, must match --pf-c-nav--Width
-  --pf-l-page__header-brand--FlexBasis--lg:                 pf-size-prem(290px);
+  --pf-l-page__header-brand--PaddingTop: var(--pf-global--spacer--md);
+  --pf-l-page__header-brand--PaddingBottom: var(--pf-global--spacer--md);
+  --pf-l-page__header-brand--PaddingLeft: var(--pf-global--spacer--xl);
+  --pf-l-page__header-brand--PaddingRight: var(--pf-global--spacer--xl);
+  --pf-l-page__header-brand--PaddingBottom--sm: var(--pf-global--spacer--md);
+  --pf-l-page__header-brand--BackgroundColor--md: var(--pf-global--BackgroundColor--dark-transparent-100); // Hardcode header brand width based on design spec, must match --pf-c-nav--Width
```

Comment:
should this comment be on the next line?

Reply:
yes! good catch.